user/templates/templates.md

```diff
@@ -218,40 +218,58 @@ Please see [How to Reinstall a Template](/doc/reinstall-template/).
-   template.
-
-    ```
-    [user@dom0 ~]$ qvm-create -l red -t <NEW_TEMPLATE> <NEW_DISPOSABLE_TEMPLATE>
-    [user@dom0 ~]$ qvm-prefs <NEW_DISPOSABLE_TEMPLATE> template_for_dispvms True
-    [user@dom0 ~]$ qvm-features <NEW_DISPOSABLE_TEMPLATE> appmenus-dispvm 1
-    [user@dom0 ~]$ qubes-prefs default-dispvm <NEW_DISPOSABLE_TEMPLATE>
-    ```
+1. Make the new or upgraded template the default template. In the App Menu, go
+   to Qubes Tools, then click on Qubes Global Settings. In the Qube Defaults
+   section, next to Template, select the new or upgraded template from the
+   drop-down list. Press OK.
+
+2. Base your [disposable templates](/doc/glossary/#disposable-template) on the
+   new template. In the App Menu, go to Qubes Tools, then click on Qube
+   Manager. In the Qube Manager, find your disposable templates. (By default,
+   they end in `-dvm`.) Right click, then click on Settings. Next to Template,
+   select your new or upgraded template from the drop-down list. Press OK.
+
+   - If you receive an error message because there are running
+     [disposables](/doc/glossary/#disposable) based on this disposable
+     template, and your only keyboard and mouse are *not* connected through a
+     [USB qube](/doc/usb-qubes/), or that USB qube is *not* a disposable, then
+     shut down all disposables based on the disposable template and try again.
+
+   - If your only keyboard or mouse *are* connected through a USB qube, and
+     that USB qube *is* a disposable, then you will have to enter a special
+     command that shuts down all of your qubes, switches the USB qube's
+     disposable template to the new template, then starts the USB qube again.
+     In order to avoid being locked out of your system, you must be very
+     careful to enter this command without typos and with the correct
+     substitutions.
+
+     In the App Menu, click on Terminal Emulator. Type the command below,
+     substituting `<SYS_USB_DISPOSABLE_TEMPLATE>` with the name of the
+     disposable template on which `sys-usb` is based, `<NEW_TEMPLATE>` with the
+     name of the new or upgraded template, and `<USB_QUBE>` with the name of
+     your USB qube. Other than these substitutions, make sure to enter the
+     command exactly as written.
+
+     ```
+     qvm-shutdown --wait --all; qvm-prefs <SYS_USB_DISPOSABLE_TEMPLATE> template <NEW_TEMPLATE>; qvm-start <USB_QUBE>
+     ```
+
+     With substitutions, your command should look similar to this example.
+     (Warning: This is just an example. Do not attempt to use it.)
+
+     ```
+     qvm-shutdown --wait --all; qvm-prefs fedora-01-dvm template fedora-02; qvm-start sys-usb
+     ```
+
+3. Base your app qubes on the new template. In the App Menu, go to Qubes Tools,
+   then click on Qubes Template Manager. Select all the templates based on the
+   old template (shown in the Current column). Next to "Change all selected
+   to," select the new or upgraded template from the drop-down list. Press OK.
```

Comment:
This works, but since user has Qube Manager open already, one can sort by template, select all with the old template, and change it from right click menu.

Reply:
Won't that require way too much clicking around if the user has a lot of templates to change? I thought that's the problem the Template Manager tool was created to address. Or am I missing something?

Reply:
> That's true, if you want to change a lot of templates (not just a lot of VMs), then Template Manager is the right tool for the job.

Hm, I don't understand. How does changing a lot of VMs not require tons of clicking without the Template Manager?

Reply:
Because you can select multiple qubes and change their template at once. If you sort by template, then select first, and then last with shift.

Reply:
Oh, I didn't realize you could do that! :exploding_head: